viewer/public/common.js

```diff
@@ -785,6 +785,42 @@ $(document).ready(function() {
     });
     return false;
   });
+  
+  //////////////////////////////////////////////////////////////////////////////////
+  // Reports
+  //////////////////////////////////////////////////////////////////////////////////
+  $("#queryReport").click(function (e) {
+    var that = this;
+    showActionsDialog({
+      submit: "Download report",
+      title: "Report - " + $(this).attr("label"),
+      input: "Filename",
+      query: "Export",
+      defaultInput: $(this).attr("label") + ".html"
+    }, function(qs, ids, filename) {
+      if (ids) {
+        qs.push({name: "ids", value: ids});
+      }
+      window.location = "queryReport/" + $(that).attr("data-filename") + "/" + filename + "?" + $.param(qs);
+    });
+    return false;
+  });
+  
+  $("#sessionReport").click(function (e) {
```

Comment:
when more then one report exists, only the first one can be used
ex: report2, report3 can't be used
report
--report1
--report2
--report3

recommend:
$(".sessionReport").click(function (e) {